docs/get-started/sumo-logic-ui-new.md

```diff
@@ -211,6 +211,86 @@ You don't have to stop there either. You can take the next step and become Sumo
+
+### I have some concerns regarding unfamiliar experiences and potential issues arising from them. How is Sumo Logic addressing these concerns to ensure a seamless transition?
+
+Understanding the challenges that come with change, we are confident that the New UI will offer our customers a notably enhanced, faster, and more seamlessly integrated experience. To facilitate a smooth transition, we have taken the following steps:
+
+* **Dogfooding**. We at Sumo Logic are the biggest customer of our own platform. Through extensive dogfooding, we've gained a deep understanding of workflows and addressed any issues that arose.
+* **Beta Testing**. We're conducting extensive beta testing with a large group of our customers. This allows us to gather feedback and address any pain points that may have been overlooked. If you'd like to participate, reach out to your Sumo Logic account executive.
+* **Opt-in General Availability (GA)**. Additionally, we will have an opt-in period lasting at least three months. During this time, users can choose to opt-in to the new experience, enabling them to become familiar with it gradually before a complete switch occurs.
+
+### I like that all Sumo Logic tabs are grouped together in one browser tab. With this change, will I have an excessive amount of Sumo Logic tabs in my browser?
+
+We're aware that the removal of in-app tabs may be the most significant change in our user workflow. We've received feedback from users, both positive and negative, regarding our tabs. While some users appreciate the convenience of having all the tabs within the app, others have questioned why we're trying to reinvent the tabs.
+
+Having in-app tabs has caused performance issues, which has been a complaint about the Sumo Logic UI. On one hand, consolidating all the tabs together reduces the number of browser tabs, but on the other hand, it can make it difficult to manage multiple sessions.
+
+If you prefer having all Sumo Logic tabs in one place - without getting mixed with other browser tabs - we suggest utilizing the [tab grouping functionality](#customize-your-environment-withtabs) offered by most browsers. By adding your Sumo Logic tab to a tab group, any new tabs opened within the Sumo Logic platform will automatically open in the same tab group. You can also collapse the tab group to reclaim valuable real estate in the browser's tab bar.
+
+### Will the New UI continue the functionality of remembering the tabs I had opened in my previous session, similar to the Classic UI?
+
+After analyzing tab usage data, we've found that a minimal portion of previously opened tabs are actively utilized by our users. Consequently, the Sumo Logic UI often remains cluttered with multiple unused tabs. With the introduction of the New UI experience, if you fail to close browser tabs from previous sessions, they will automatically reload upon login.
+
+<img src={useBaseUrl('img/get-started/tab-reload.gif')} alt="tab-reload.gif" />
```

Comment:
This gif needs to move up into the previous section since it shows tab grouping.